### PR TITLE
Changes of edge graph

### PR DESCRIPTION
**Pull request by MeikeWeiss**
I thought it would make sense to have a symmetric edge digraph like for the face graph. Furthermore, there are some other minor changes.

### CODE REVIEW

File: gap/PolygonalComplexes/graphs.gd [hunk 1]
```diff
@@ -799,20 +799,32 @@ DeclareOperation( "OnEdgeFacePaths",
 #! edge graph are the vertices of <A>complex</A> and for every edge in
 #! <A>complex</A> there is a corresponding edge in the edge graph.
 #!
-#! TODO example
+#! Since the edges in Digraphs are directed but the face graph is undirected,
```
Comment (TomGoertzen) [inline gap/PolygonalComplexes/graphs.gd:802]: Shouldn't it say: "the edge graph is undirected" :)?

**Review by TomGoertzen — CHANGES_REQUESTED** [review]

**Review by TomGoertzen — APPROVED** [review]

**Review by TomGoertzen — APPROVED** [review]
Looks good

**Review by ReymondAkpanya — CHANGES_REQUESTED** [review]
this looks fine :)

**Comment by MeikeWeiss** [comment]
Thanks! Can you please approve the pull request then?

**Review by ReymondAkpanya — APPROVED** [review]

**Pull request merged**